### PR TITLE
Kk/notification improvements

### PR DESCRIPTION
**Pull request by kkostalkowicz**

### CODE REVIEW

File: src/transactionNotifier/TransactionNotifierView.tsx [hunk 1]
```diff
@@ -82,14 +86,14 @@ export function describeTxStatus(tx: TxState) {
...
         <>
-          Propagating <Timer start={tx.broadcastedAt} />
+          Unconfirmed <Timer start={tx.broadcastedAt} />
         </>
       );
     case TxStatus.WaitingForConfirmation:
       return (
-        <>
+        <span className={styles.hilight}>
           Unconfirmed <Timer start={tx.broadcastedAt} />
```
Comment (maciejka) [inline src/transactionNotifier/TransactionNotifierView.tsx]: Shouldn't link to etherscan be provided here? That was @geronimo's idea afaik.

Comment (kkostalkowicz) [reply src/transactionNotifier/TransactionNotifierView.tsx]: That was not requested in E2D-396

Comment (maciejka) [reply src/transactionNotifier/TransactionNotifierView.tsx]: Now it is. ;-)

File: src/transactionNotifier/TransactionNotifier.scss [hunk 2]
```diff
@@ -64,7 +64,9 @@
 .description {
   color: $grey-light;
 }
-
+.hilight {
```
Comment (peculiarity) [inline src/transactionNotifier/TransactionNotifier.scss]: highlight

File: src/transactionNotifier/TransactionNotifier.scss [hunk 3]
```diff
@@ -64,7 +64,9 @@
 .description {
   color: $grey-light;
 }
-
+.link {
+  color: #80D8FF;
```
Comment (peculiarity) [inline src/transactionNotifier/TransactionNotifier.scss]: Can you extract the color in the variable.scss? I use colorhexa.com for proper naming :)

Comment (kkostalkowicz) [reply src/transactionNotifier/TransactionNotifier.scss]: I always search for colors in variables.scss first, but for some reason it did not work this time...

**Pull request merged**